src/store/metadata/stub/content/editions/ons-energy-use-fossil-fuels-by-fuel-type-and-industry.json

```diff
@@ -0,0 +1,101 @@
+{
+  "@context": "https://staging.idpd.uk/#ns",
+  "@id": "https://staging.idpd.uk/datasets/ons-energy-use-fossil-fuels-by-fuel-type-and-industry/editions",
+  "@type": "hydra:Collection",
+  "title": "Energy use: fossil fuels by fuel type and industry",
+  "editions": [
+    {
+      "@id": "https://staging.idpd.uk/datasets/ons-energy-use-fossil-fuels-by-fuel-type-and-industry/editions/2023-06",
+      "@type": "dcat:Dataset",
+      "in_series": "https://staging.idpd.uk/datasets/ons-energy-use-fossil-fuels-by-fuel-type-and-industry",
+      "identifier": "2023-06",
+      "title": "Energy use: fossil fuels by fuel type and industry",
+      "summary": "The UK's fuel use by industry (SIC 2007 group - around 130 categories) and type (coal, natural gas, petrol, diesel oil for road vehicles (DERV), fuel oil, gas oil, aviation fuel and other), 1990 to 2021. The dataset excludes biofuels and waste.",
+      "description": "The UK's fuel use by industry (SIC 2007 group - around 130 categories) and type (coal, natural gas, petrol, diesel oil for road vehicles (DERV), fuel oil, gas oil, aviation fuel and other), 1990 to 2021. The dataset excludes biofuels and waste.",
+      "publisher": "https://staging.idpd.uk/office-for-national-statistics",
+      "creator": "office-for-national-statistics",
+      "contact_point": {
+        "name": "Environment Accounts",
+        "email": "mailto:environment.accounts@ons.gov.uk"
+      },
+      "topics": [
+        "https://staging.idpd.uk/topics/climatechange",
+        "https://staging.idpd.uk/topics/carbonemissions"],
+      "frequency": "irregular",
```

Comment:
It's not released regularly? Like annually or something?

Reply:
oversight; from an existing example. 
frequency changed to `annual`